requirements.txt

```diff
@@ -27,7 +27,9 @@ mkdocs-material==9.1.15
 mkdocstrings[python-legacy]==0.22.0
 netaddr==0.8.0
 Pillow==9.5.0
-psycopg2-binary==2.9.6
+psycopg==3.1.9
+psycopg-binary==3.1.9
+psycopg-pool==3.1.7
```

Comment:
Are all three of these necessary?

Reply:
[Found this article](https://learndjango.com/tutorials/psycopg3-binary-and-django-42-installation-quick-t) which partially answers my question. And it seems that pooling has also been split out from the main `psycopg` package.

Reply:
made to one line